README.md

```diff
@@ -1309,6 +1310,10 @@ suite, just run `npm test`.
 
 You can also run the tests with verbose output using `npm test-verbose`
 
+## Migrating to `cloudant-node-sdk` library
+We have a newly supported Cloudant Python SDK named [cloudant-node-sdk](https://github.com/IBM/cloudant-node-sdk).
```

Comment:
```suggestion
We have a newly supported Cloudant Node.js SDK named [cloudant-node-sdk](https://github.com/IBM/cloudant-node-sdk).
```

Reply:
Nice catch!
However, I would prefer to use just `Node`, because `ts` also supported.

Reply:
![image](https://user-images.githubusercontent.com/9086834/104914112-f1175b80-598e-11eb-9d0e-0389b0d4e5bc.png)

I get that `js` might be slightly misleading there, but the SDK is called `IBM Cloudant Node.js SDK`. Same thing for the platform itself. It is called `node.js`  ( https://nodejs.org/en/ )

Not a big deal though. Just being pedantic.

Reply:
This was new for me. In this case I am okay with the suggestion. Changed in 8b5920f